rpc/wrtc_client.go

```diff
@@ -203,7 +203,7 @@ func dialWebRTC(
 				},
 			})
 		})
-		return err
+		return filterEOF(err, logger)
```

Comment:
this is in `sendDone()` where I imagine that the caller / answerer tries to `sendDone()` but the signaling server has already severed the connection leading to an EOF